packages/odyssey-react/src/components/List/List.theme.ts

```diff
@@ -0,0 +1,24 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import type { ThemeReducer } from "@okta/odyssey-react-theme";
+
+export const theme: ThemeReducer = (theme) => ({
+  DescriptionGridGapColumns: theme.SpaceRemM,
+  DescriptionGridGapRows: theme.SpaceRemXs,
+  MarginBlockEnd: theme.SpaceRemM,
+  MarginBlockStart: 0,
+  MarginInline: 0,
```

Comment:
we've been prefixing these with `Space` (same for padding below). These can and will be used alongside global token names that use that pattern so we want the API to feel consistent!

Reply:
update in e9bb90c